src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -606,6 +612,27 @@ public RowMap getRow() throws Exception {
 					} else {
 						processQueryEvent(event);
 					}
+					if (sql.startsWith(BinlogConnectorEvent.XASTART) || sql.startsWith(BinlogConnectorEvent.XABEGIN)) {
```

Comment:
let's move all of these `if` statements into the above `if` block.

Reply:
OK!The method of processQueryEvent is deal with some DDL event.
I will move all of these if statements into the above if block.